docs/understand/model.md

```diff
@@ -18,77 +18,69 @@ defines the common concepts of lakeFS.
-a _merge commit_. Currently lakeFS only supports merge commits with two parents.
+Using commits, you can view a [repository](#repository) at a certain point in its history and you're guaranteed that the data you see is exactly as it was at the point of committing it.
 
-#### Identifying commits
-{: .no_toc }
+These commits are immutable "checkpoints" containing all contents of a repository at a given point in time. This is very similar to commits in Git.
 
-A commit is identified by its _commit ID_, a digest of all contents of the commit. Commit IDs are by nature long,
-so you may use a unique prefix to abbreviate them. A commit may also be identified by using a textual definition,
-called a _ref_. Examples of refs include tags, branch names, and expressions.
+Each commit contains metadata - the committer, timestamp, a commit message, as well as arbitrary key/value pairs you can choose to add.
 
-### Branches
-_Branches_ are similar in concept to [Git branches](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging){:target="_blank"}.
-It is a mutable pointer to a commit and its staging area (see [below](#staging-area)). A branch in lakeFS is a consistent snapshot of the entire repository,
-which is isolated from other branches and their changes.
 
-Example branches:
+  **Identifying Commits**<br/><br/>
+  A commit is identified by its _commit ID_, a digest of all contents of the commit. <br/>
+  Commit IDs are by nature long, so you may use a unique prefix to abbreviate them. A commit may also be identified by using a textual definition, called a _ref_. <br/><br/>
+  Examples of refs include tags, branch names, and expressions.
+{: .note }
+
 
-* `main`, the trunk.
-* `staging`, maybe ahead of `main`.
-* `dev:joe-bugfix-1234` for Joe to fix issue 1234.
+### Branches
 
-### Staging Area
+Branches in lakeFS allow users to create their own "isolated" view of the repository.
 
-The _staging area_ is where your changes appear before they are committed.
-Unlike Git, every branch in lakeFS has its own staging area.
-Uncommitted changes are visible to all users with read permissions, for example using the _Uncommitted Changes_ tab in the UI.
+Changes on one branch do not appear on other branches. Users can take changes from one branch and apply it to another by [merging](#merge) them.
 
-When you commit these changes, a new commit is added to the commit history, and the changes disappear from the staging area.
-This operation is atomic: readers will either see all your committed changes or none at all.
+Under the hood, branches are simply a pointer to a [commit](#commits) along with a set of uncommitted changes.
 
 
 ### Tags
 
-A _tag_ is an immutable pointer to a single commit. Tags have readable names. Since tags
-are commits, a repository can be read from any tag. Example tags:
+Tags are a way to give a meaningful name to a specific commit.
+
```

Comment:
would it make sense to add a sentence about tags here? Something like:
`they enable you to do a quick search and cross-reference ... `

Reply:
Added